tests/sentry/integrations/bitbucket/test_issues.py

```diff
@@ -260,3 +260,27 @@ def test_get_link_issue_config(self):
                 'help': ('Leave blank if you don\'t want to '
                          'add a comment to the Bitbucket issue.'),
             }]
+
+    @responses.activate
+    def test_get_issue(self):
```

Comment:
Isn't the test for `create_issue`?

Reply:
Unfortunately no. I didn't see a test for it at all.